dipy/segment/tests/test_feature.py

```diff
@@ -284,6 +285,26 @@ def extract(self, streamline):
     d2 = metric.dist(features1, features2)
     assert_equal(d1, d2)
 
+    class ArcLengthFeature(dipymetric.Feature):
+        def infer_shape(self, streamline):
+            if sys.version_info > (3,):
```

Comment:
Would this do it? https://github.com/MarcCote/dipy/commit/13687569f666d499f84735c97910cd4312ba41b9